include/exadg/incompressible_navier_stokes/user_interface/enum_types.h

```diff
@@ -676,6 +676,49 @@ enum class TurbulenceEddyViscosityModel
 std::string
 enum_to_string(TurbulenceEddyViscosityModel const enum_type);
 
+/**************************************************************************************/
+/*                                                                                    */
+/*                            GENERALIZED NEWTONIAN MODELS                            */
+/*                                                                                    */
+/**************************************************************************************/
+
+/*
+ *  Generalized Newtonian models, see, e.g., Galdi et al., 2008
+ *  ("Hemodynamical Flows: Modeling, Analysis and Simulation").
+ *  With
+ *  y    = sqrt(2*sym_grad_velocity : sym_grad_velocity)
+ *  e_oo = generalized_newtonian_kinematic_viscosity_lower_limit
+ *  e_0  = generalized_newtonian_kinematic_viscosity_upper_limit
+ *  k    = generalized_newtonian_kappa
+ *  l    = generalized_newtonian_lambda
+ *  a    = generalized_newtonian_a
+ *  b    = generalized_newtonian_b
+ *  we have the apparent viscosity
+ *  viscosity = e_oo + (e_0-e_oo) * [k + (l*y)^a]^[(n-1)/a]
+ */
```

Comment:
I tend to say that this explanation should appear in the implementation, not in the enum declaration file. But I do not have a strong opinion here.